domoticz.sh

```diff
@@ -11,20 +11,34 @@
 
 # Do NOT "set -e"
 
+SCRIPT=$(readlink -f "$0")
```

Comment:
why not the 'pwd' command ?
for the local.settings, maybe better to use /etc/domoticz.conf ?

Reply:
It's not `pwd` because I'm not interested in what directory the script is being run from. It's interested in where the script is located. 

Sometimes they might be the same. But I know when I run it manually, they're usually not, so depending on the actual location is better.

Reply:
+1, thanks for the clarification!

Reply:
How about the use of /etc/domoticz.conf?